### ISSUE TITLE
dotnet-sdk-8.0 is not available on Fedora 38

### ISSUE THREAD

**Issue by marcinjahn** [dotnet/docs#38366]
### Type of issue

Code doesn't work

### Description

The article says that Fedora 38 is supported for .NET 8 installation. The following installation instruction is provided:

```sh
sudo dnf install dotnet-sdk-8.0 
```

It results in:

> Last metadata expiration check: 2:07:15 ago on Wed 22 Nov 2023 07:28:35 AM CET.
No match for argument: dotnet-sdk-8.0
Error: Unable to find a match: dotnet-sdk-8.0

It seems that `dotnet-sdk-8.0 ` is not available in default Fedora 38 repositories, which is unexpected. The article makes it seem like it should be there.
Is it just too early, and the package will be available some time later?

### Page URL

https://learn.microsoft.com/en-us/dotnet/core/install/linux-fedora

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/core/install/linux-fedora.md

### Document Version Independent Id

2e9467f4-8572-c8e4-6a10-69faf14ce46c

### Article author

adegeo

### Metadata

* ID: 38a30294-c9ae-4189-26d7-ee7db615c487 
* Product: **dotnet-fundamentals**
* Technology: **dotnet-install**

**Comment by ildar-shaymukhametov** [comment]
Same here
I had to install windows to use net8.0

**Comment by marcinjahn** [comment]
> Same here I had to install windows to use net8.0

That’s quite an extreme workaround :)

**Comment by DocNITE** [comment]
Same. Where i can download sdk 8.0 for fedora?

**Comment by ildar-shaymukhametov** [comment]
I do not know. I think you should install windows till the issue is resolved

**Comment by christianfosli** [comment]
I think we need to wait for this https://bugzilla.redhat.com/show_bug.cgi?id=2244633 to be approved first.

**Comment by MrSmoke** [comment]
@atrauzzi thanks, I was just about to do the same :pray:

**Comment by adegeo** [comment]
Sorry for the confusion here.

@omajid when is .NET 8 expected for 38?

**Comment by adegeo** [comment]
I'll add a note to the article that says it may not yet be available.

**Comment by sitepodmatt** [comment]
>
.NET 8 was released on November 14, 2023. It may take time for the packages to appear in the package manager feeds

This seems utterly bizarre way to resolve this issue.  Surely we can say it's not available yet, and then update when it is. It's easy to infer that this might just be a repo sync disclaimer, if it's not available on 14th try the 15th etc..   The package isn't even approved yet and still in review - albeit out of Microsoft control.  There so much opportunity here to be less ambiguous.

**Comment by kstatz12** [comment]
https://github.com/hensou/asdf-dotnet for now ive been using this as a workaround. I also use it to manage node versions across multiple projects and its fantastic.

**Comment by sitepodmatt** [comment]
> https://github.com/hensou/asdf-dotnet for now ive been using this as a workaround. I also use it to manage node versions across multiple projects and its fantastic.

That's probably a viable option as a workaround if urgent,  but many of us are getting a overwhelmed with having multiple package managers and local vs global debate, by the necessity of one not supporting everything  I'd rather just choose my dev dependencies more carefully (or wait a little longer for 8.0 in this case).   nvm, sdkman, brew, pip, rvm, virtualenv, bob (for neovim), yikes more than I thought accumulated. 

They still need to address the entirely ambiguous and inadequate messaging here.  _may_ = is not

**Comment by shaimildar** [comment]
I used this guide to install net8
https://developercommunity.visualstudio.com/t/Fedora-39-NET-SDK-80-not-found-in-repo/10524729?space=8&q=nuget&sort=newest

**Comment by sitepodmatt** [comment]
That's a better option than asdf imo,  but I'm just going to wait it out now, I just hope next year they don't prematurely tell Fedora users 9.0 is available, and then bizarrely just introduce the phrase 'may not' be as some kind of fix ~3 weeks later, kind of ridiculous all round rather than just removing the page - or headlining with not currently available (WIP) and then updating when it is.

**Comment by sitepodmatt** [comment]
Possibly just following Ubuntu, it certainly used to be that way then I believe Canonical and Microsoft partnered for dotnet 6.0 around 2-3 years ago.  If you add it now you have need to somehow add precendence to favor the MS repos for 6.0/7.0 updates, also not sure what happens to existing 6.0/7.0 installed for the main repos?

**Comment by omajid** [comment]
Sorry, I was away on vacation for the last few weeks.

> when is .NET 8 expected for 38?

The Fedora .NET 8 package review was stalled, but seems to be moving now. I plan to build .NET 8 in Fedora as soon as a the package review is completed. That could be a week, or it may be longer depending on how long the review goes.

**Comment by christianfosli** [comment]
Looks like it is available through the fedora dnf repos now 😊

**Comment by omajid** [comment]
> Looks like it is available through the fedora dnf repos now 😊

Yes! It's available for Rawhide and Fedora 39 already. Fedora 38 is [in testing](https://bodhi.fedoraproject.org/updates/FEDORA-2023-eefadf07e8) and should be available in a few days.

**Comment by omajid** [comment]
@atrauzzi Fedora has 2 forms of testing for updates: automated and manual.

The automated testing failed but a packager (like me) can waive it with the click of a button.

However, unless a user grabs the update from the testing repo and gives it karma on https://bodhi.fedoraproject.org/updates/FEDORA-2023-eefadf07e8, the update will be stuck in testing repo for a week before reaching users.
